addons/website/views/snippets/snippets.xml

```diff
@@ -411,9 +411,48 @@
          data-selector='.accordion > .card'
          data-drop-in='.accordion:has(> .card)'/>
 
+    <div data-js="MasonryLayout"
```

Comment:
Options which are only for one snippet should be in the file where the snippet is defined